packages/stream-model-instance-handler/src/model-instance-document-handler.ts

```diff
@@ -194,9 +194,9 @@ export class ModelInstanceDocumentHandler implements StreamHandler<ModelInstance
     }
 
     const oldContent = state.content ?? {}
-    const newContent = jsonpatch.applyPatch(oldContent, payload.data).newDocument
+    const newContent = jsonpatch.applyPatch(oldContent, payload.data, undefined, false).newDocument
     const modelStream = await context.loadStream<Model>(metadata.model)
-    await this._validateContent(context, modelStream, newContent, false, payload)
+    await this._validateContent(context, modelStream, newContent, false, payload, oldContent)
```

Comment:
Couldn't `state.content` be passed directly here, having the logic handle if it's `null`?

Reply:
yes it could, but the oldContent line already handles if state.content its null, what would be the benefit of doing that here?

Reply:
I just think the `null` value should be provided directly, no need to provide an empty object as fallback, so the following logic can do the null check.

Reply:
Rather than relying on the content being `null`, which feels like an indirect check of what we want and maybe not entirely future-proof (what if stream content could be explicitly set to null in the future?), could we instead check that the stream has a set account relation and that this is the first data commit?  That feels like the actual rule that should allow content to be set for the first time.

Reply:
you mean checking the state.log and confirm this is the first data commit? or do you mean something else?